src/docs/releases/2.0.0.md

```diff
@@ -246,6 +246,11 @@ Here are the updated signatures:
 
 These adjustments ensure compatibility and adherence to the latest conventions within the `SectionDisplayDriver` class.
 
+### GraphQL Module
+
+The GraphQL schema may change, because fields are now always added to the correct part. Previously, additional fields may have been added to the parent content item type directly. 
```

Comment:
```suggestion
The GraphQL schema may change because fields are now always added to the correct part. Previously, additional fields may have been added to the parent content item type directly. 
```